src/Http/Controllers/CourseAPIController.php

```diff
@@ -86,16 +87,18 @@ public function store(CreateCourseAPIRequest $request)
         return $this->sendResponse($course->toArray(), 'Course saved successfully');
     }
 
-    public function show($id)
+    public function show($id, GetCourseAPIRequest $request)
```

Comment:
eg https://lms-demo.stage.etd24.pl/courses/6 

this is a preview, summary endpoint accessible to anybody. Program/Curriculum/Sylabus is just available to ones that own the,